conan/internal/api/config/config_installer.py

```diff
@@ -127,13 +128,12 @@ def _process_file(directory, filename, config, cache_folder, folder):
         output.info("Copying file %s to %s" % (filename, target_folder))
         _filecopy(directory, filename, target_folder)
 
-
 def _process_folder(config, folder, cache_folder, ignore=None):
     if not os.path.isdir(folder):
         raise ConanException("No such directory: '%s'" % str(folder))
     if config.source_folder:
         folder = os.path.join(folder, config.source_folder)
-    conanignore_path = os.path.join(folder, '.conanignore')
+    conanignore_path = find_file_walk_up(folder, ".conanignore", end=os.path.abspath(config.uri))
```

Comment:
This is confusing. ``config.uri`` could be a URL, isn't it? So this might fail?
It seems that the original ``folder`` argument value is what should be used.